ui/public/locales/en/translation.json

```
@@ -1182,6 +1166,22 @@
       "loading": "loading...",
       "optionDoesNotExist": "{{namespace}} (does not exist)"
     },
+    "variableForm": {
+      "name": {
+        "label": "Name",
+        "placeholder": "variable-name",
+        "nameAlreadyExists": "The name already exists"
+      },
+      "mimeType": {
+        "label": "Mimetype",
+        "empty": "Mimetype is empty",
```

```suggestion
        "unspecified": "Mimetype is unspecified",
```

Not a big issue at all, but the wording sounds a bit funny. In the component it makes sense to have a property `mimeTypeIsEmpty` because we're dealing with an empty string. But semantically, it makes more sense to read the mime type is "unspecified".

---

Good catch, I changed it in 68913f5a8a1c3c8732a36cfbe11713113ff4fb2b